src/flow.js

```diff
@@ -108,13 +111,18 @@ export default class Flow {
     // workout what the new previous step should be.
     if (this._prevStepIsNotDefinedOrEqualsCurrentStep(state)) {
       if (this._prevStepIsInFlow(state)) {
+        console.log(state)
```

Comment:
Unless they are necessary, remove `console.log` from code - there are a few of them between lines 114 and 125.